Tests/ACKLocalizationCoreTests/LocRowTests.swift

```diff
@@ -81,4 +81,14 @@ final class LocRowTests: XCTestCase {
         let locRow = LocRow(key: "pos_arg_key", value: "%1$@ people will arrive in %2$@ minutes")
         XCTAssertEqual(#""pos_arg_key" = "%1$@ people will arrive in %2$@ minutes";"#, locRow.localizableRow)
     }
+
+    func testForDuplicatedKeys() throws {
```

Comment:
please move it to the `ACKLocalizationTests`, this has nothing to do with `LocRow`. Also add test for success path (one duplicate keys)